src/yarpmanager/src-builder/arrow.cpp

```diff
@@ -57,35 +57,29 @@ const qreal Pi = 3.14;
 
 
 
-Arrow::Arrow(BuilderItem *startItem, BuilderItem *endItem,int id, Manager *safeManager,
-             bool isInApp,bool editingMode, BuilderItem *parent)
-    : BuilderItem(parent),textLbl("",this)
+Arrow::Arrow(BuilderItem *startItem, BuilderItem *endItem,int _id, Manager *safeManager,
+             bool isInApp,bool editingMode, BuilderItem *parent) :
+    BuilderItem(parent),
+    manager(safeManager),
+    externalSelection(false),
+    editingMode(editingMode),
+    connected(false),
+    myStartItem(startItem),
+    myEndItem(endItem),
+    myColor(Qt::black),
+    textLbl("",this),
+    textWidth(0),
+    id(_id),
+    firstTime(true)
 {
-    sigHandler = new ItemSignalHandler();
     itemType = ConnectionItemType;
-    connected = false;
-    myStartItem = startItem;
-    myEndItem = endItem;
-    this->manager = safeManager;
+    nestedInApp = isInApp;
+    sigHandler = new ItemSignalHandler();
```

Comment:
Is there a reason why you didn't move these in the initialization list as well?

Reply:
Yes they're base class members and i could not init them in the list.